.github/workflows/helm-publish.yaml

```diff
@@ -0,0 +1,63 @@
+name: Helm publish
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  CHARTS_REPOSITORY: ${{ github.repository_owner }}/charts
+  CHART_NAME: etcd-operator
+
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4.1.1
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4.1.0
+        with:
+          version: 'v3.14.3'
+
+      - name: Get latest tag
+        run: |
+          git fetch --tags
+          TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          echo "Latest tag on current commit is $TAG"
+          echo "LATEST_TAG=${TAG}" >> $GITHUB_ENV
+          echo "LATEST_TAG_TRIMMED_V=${TAG#v}" >> $GITHUB_ENV
```

Comment:
I don't think it's a good idea to run pipeline on tag `v1.2.3` and create helm build for another

Reply:
Got it, I'll rethink this approach.

Reply:
Now it's using `${{ github.ref_name }}`